### PR TITLE
added model training and exercise

### PR DESCRIPTION
**Pull request by Enrikokokoko**

### CODE REVIEW

File: src/training/schemas/training.schema.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import { Prop, Schema, SchemaFactory } from "@nestjs/mongoose";
+import mongoose from "mongoose";
+import { User } from "src/user/schemas/user.schema";
+
+@Schema()
+export class Training {
+    @Prop({ type: mongoose.Schema.Types.ObjectId, ref: 'User'})
+    user: User;
+
+    @Prop({ type: Date, startAt: '2023-06-02'})
```
Comment (VladOSTAFI) [inline src/training/schemas/training.schema.ts:10]: Why do we need `startAt: '2023-06-02'`?

File: src/training/schemas/training.schema.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    date: number;
+
+    @Prop({ type: Date })
```
Comment (VladOSTAFI) [inline src/training/schemas/training.schema.ts:13]: I guess we can set default value here.
It should be something like @Prop({ type: Date, default: Date.now }).
Research how to do it please.

**Review by VladOSTAFI — CHANGES_REQUESTED** [review]
Please run `npm run prettify` script before next push

File: src/training/schemas/training.schema.ts (hunk 1 above)
Comment (Enrikokokoko) [reply src/training/schemas/training.schema.ts:10]: It's like example cus i didn't know how to create Prop for date

Comment (VladOSTAFI) [reply src/training/schemas/training.schema.ts:10]: Can we just leave `@Prop({ type: Date })`?

**Review by VladOSTAFI — APPROVED** [review]

**Pull request merged**